fennel/datasets/datasets.py

```diff
@@ -1538,26 +1565,32 @@ def sync_validation_for_pipelines(pipelines: List[Pipeline], ds_name: str):
     This validation function contains the checks that are run just before the sync call.
     It should only contain checks that are not possible to run during the registration phase/compilation phase.
     """
-    versions = set()
-    for pipeline in pipelines:
-        if pipeline.version in versions:
-            raise ValueError(
-                f"Pipeline {pipeline.fqn} has the same version as another pipeline in the dataset."
-            )
-        versions.add(pipeline.version)
-
-    found_active = False
-    for pipeline in pipelines:
-        if pipeline.active and found_active:
-            raise ValueError(
-                f"Multiple active pipelines are not supported for dataset {ds_name}."
-            )
-        if pipeline.active:
-            found_active = True
-    if not found_active and len(pipelines) > 1:
-        raise ValueError(
-            f"No active pipeline found for dataset {ds_name}. Please mark one of the pipelines as active by setting `active=True`."
-        )
+    if len(pipelines) > 1:
```

Comment:
if len(pipelines) <= 1:
return

.. rest of code ..

This prevents indenting the whole code

Reply:
changed this.